apps/andi/priv/static/css/app.css

```diff
@@ -0,0 +1,137 @@
+/* Includes some default style for the starter application.
```

Comment:
This css file appears to be the same as phoenix.css. You're probably already thinking about this but, I think we'll want to wipe out this file with the exception perhaps of Milligram (Though it's odd to me that Milligram was just copied into here instead of being imported as a dependency)